examples/Workflows/README.md

```diff
@@ -1,3 +1,8 @@
 # Workflow examples
 
- * [Pipeline/Sequential workflow using densenet161 image classifier example](Densenet%20Image%20Classifier)
\ No newline at end of file
+The following links provide examples on how to implement workflows with different models. The workflow feature can be used for serving an ensemble of models and python functions through workflow APIs. A workflow is executed as a DAG where the nodes can be either model (MAR files) or functions specified in the workflow handler file. The DAG need not contain additional functions if not required. Typically, the function nodes are used for processing or augmenting intermediate data or aggregating data from multiple nodes. Currently, only sequential workflow is supported fully.
```

Comment:
@maaquib It will help to references the examples here to clarify in what scenarios, preprocessing is not needed in workflow handler (e.g in the nmt back translation was not required)/ it can be handled by model handler, or in case of parallel model ensemble like dual translation, it is required to have preprocessing as single node for the start/ CV example preprocessing node enables us to share the original input with arbitrary nodes in DAG.